lib/monkey-mailer/adapters/mandrilapi.rb

```diff
@@ -59,6 +56,27 @@ def send_email(email)
         raise MonkeyMailer::DeliverError.new("Mandril response.code not equal to 200") unless response.code.to_i == 200
         puts "Response #{response.code} #{response.message}: #{response.body}"
       end
+
+      def parse_recipients(recipients_list, names_list = "", type = :to)
+        recipients = recipients_list.split(",")
+        names      = names_list.to_s.split(",")
+        to_list    = []
+
+        raise RuntimeError.new("No recipients specified") if recipients.empty?
+
+        if names.any? && (names.count != recipients.count)
+          raise RuntimeError.new("Recipients and Names lists don't match")
+        end
+
+        recipients.each_with_index do |recipient, idx|
+          to_list << { "email" => recipient.strip,
```

Comment:
Related to the usage of `to_s` above, isn't defensive programming to use `strip` here?